app/controllers/publishers/vacancies_controller.rb

```diff
@@ -3,6 +3,9 @@ class Publishers::VacanciesController < Publishers::Vacancies::BaseController
   before_action :invent_job_alert_search_criteria, only: %i[show preview]
 
   def show
+    redirect_to organisation_job_review_path(vacancy.id) unless vacancy.published?
+
+    validate_all_steps
```

Comment:
So if I have completed half the steps and go for a lunch break and come back and log in again, and go to this draft job, I am greeted with a long list of all the fields I have not yet filled in. Right? This seems overly aggressive.

Reply:
We have since removed this redirect, but we essentially do the same thing. We link to the show page from the dashboard and just display the errors on here. 

We ran this by Jesse and he seemed happy with it :)

Reply:
As long as Jesse is on board that's cool